brambling/forms/organizer.py

```diff
@@ -4,12 +4,21 @@
 from django.utils.crypto import get_random_string
 import floppyforms.__future__ as forms
 
-from brambling.models import (Attendee, Event, Item, ItemOption, Discount,
-                              ItemImage, Transaction, CustomForm,
-                              CustomFormField, Order, Organization, SavedReport,
-                              DwollaAccount, UNAMBIGUOUS_CHARS,
-                              OrganizationMember, BoughtItem)
-from brambling.payment.core import LIVE, TEST
+from brambling.models import Attendee
+from brambling.models import BoughtItem
+from brambling.models import CustomForm
+from brambling.models import CustomFormField
+from brambling.models import Discount
+from brambling.models import Event
+from brambling.models import Item
+from brambling.models import ItemImage
+from brambling.models import ItemOption
+from brambling.models import Order
+from brambling.models import Organization
+from brambling.models import OrganizationMember
+from brambling.models import SavedReport
+from brambling.models import Transaction
+from brambling.models import UNAMBIGUOUS_CHARS
```

Comment:
Yes, it's easier to work with in the long run because there's never a transition between the two. The git history is much cleaner and it's only a little more verbose.